exporter/prometheusremotewriteexporter/factory.go

```diff
@@ -96,7 +96,7 @@ func createDefaultConfig() configmodels.Exporter {
 	}
 }
 
-func validateAndSanitizeExternalLabels(cfg *Config) error {
+func ValidateAndSanitizeExternalLabels(cfg *Config) error {
```

Comment:
Comment? Why do you need to export this?

Reply:
Comment added

Reply:
Still not answering why is needed

Reply:
Does the new comment make more sense?

Reply:
Just move it to the exporter then, I was guessing that is what you do, so please call it in the exporter ctor.

PS: we need to talk about wrapping the exporters. We don't want to expose that (probably we will not change it now, but something that needs to happen).

Reply:
I guess you do that to change the default Config, if that is the case you can simply wrap the factory into a new factory that implements only the default Config method

Reply:
The new factory I am creating in the wrapper involves more changes than just the default config. I am extending the config, and within the `createMetricsExporter()` of the wrapper I am modifying a parameter that gets passed into the function `NewPrwExporter()`

Whenever you have time, maybe for full context can you review my linked PR in the contrib repo that implements this wrapper? you can see the factory there

Reply:
Maybe it would be better to move some more logic into the wrapper instead of this so your opinions there would be valuable

Reply:
The PR in question: open-telemetry/opentelemetry-collector-contrib#1497

Reply:
Can we do this "I can also move this function to be called within the already exported function NewPrwExporter() if that is preferred"

Reply:
👍  will do